server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -65,8 +73,20 @@ export class DeliLambdaFactory
 		private readonly serviceConfiguration: IServiceConfiguration,
 		private readonly restartOnCheckpointFailure: boolean,
 		private readonly kafkaCheckpointOnReprocessingOp: boolean,
+		private readonly invalidDocumentOrSessionCacheEnabled: boolean = false,
+		private readonly invalidDocumentCacheExpiryInMs: number = 1000 * 60 * 60 * 12,
+		invalidSessionCacheExpiryInMs: number = 1000 * 10,
+		maxCacheSize: number = 10000,
```

Comment:
I see you have config for these parameters, but why they are hard coded here? Can these param configurable?

Reply:
This is just a convenient way that allows to not specify values for these params in places where having specific configured values is not important (i.e. in unit tests).